lib/perl/Genome/Config/Command/Get.pm

```diff
@@ -0,0 +1,26 @@
+package Genome::Config::Command::Get;
+
+use strict;
+use warnings;
+
+use Genome qw();
+use Genome::Config qw();
+
+class Genome::Config::Command::Get {
+    is => 'Command',
```

Comment:
For some reason I thought `Command` was the latest version not `V1`.  I'll switch it.